_2017_pages/cfp.html

```diff
@@ -15,102 +15,273 @@ <h2 class="first">Key Dates</h2>
       </p>
 
       <h3 class="first">Synchronous Conference Day (T-0 Liftoff!)</h3>
 
       <p class=normal-text><b>Dress your best and meet all the folks </b> </br>
         This day will look more like the open source conferences you've loved in the past.
-        As an attendee this day will consist of Lightning Talks, Birds of a Feather discussion
-        sessions, and discussion sessions from speakers.
-        As a speaker we'll ask that you be able to answer questions live for folks that are on the call.
+        As an attendee this day will likely consist of Lightning Talks, Birds of a Feather discussion
+        sessions, and discussion sessions from speakers. We'd like to have speaker available answer
+        questions live for folks in sessions as well.
         The exact timing is TBD, but we plan to find time zones that are convenient for everyone
-        and may end up doing two synchronous sessions in the same calendar day
-        to meet our global community's localities.
+        and may end up doing two synchronous sessions in the same calendar day to meet our
+        global community's localities.
+      </p>
+      <h2 class="first">Presentation formats</h2>
+      <div id="talks" class="section">
+      <h3 class="first">Talks</h3>
+      <p class=normal-text>
+        Talks will consist of a 30 to 40 minutes video. As pre-recorded videos, talks are
+        expected to be polished and crisp.
+      </p>
+      <p>
+        All the content shown in the video (slides, code, plots...) and language should
+        be professional, comprehensible and follow PyMCon code of conduct.
+        A successful talk does not have to be about a novel, cutting-edge technology
+        topic but should be distinctive and have the ability to keep the attention of
+        the audience. We will provide a platform for audience discussions and Q&A.
+        Accepted speakers are asked to be online the day of for at least one session of
+        Q&A to answer questions live, which is 20 minutes long;
+        of course, they are free to participate in the discussions with the audience
+        outside of this Q&A session as well.
+      </p>
+      </div>
+
+      <div id="tutorials" class="section">
+      <h3 class="first">Tutorials</h3>
+      <p class=normal-text>
+        Tutorials are dedicated to introducing, promoting, and advancing specific
+        techniques/practices in computational Bayesian statistics with PyMC. They should
+        cover a topic more in depth than a traditional talk.
+        Tutorials will consist of a 60 minutes video and extra material for attendees to
+        engage and experiment with the contents of the tutorial.
+      </p>
+      <p>
+        Tutorial facilitators are responsible for preparing interactive and carefully
+        designed materials, including hands-on exercises that help reinforce the concepts
+        discussed in their topics. We will provide the platform (most likely Slack) in
```

Comment:
Im going to change this to discourse since we for sure are going to have that one where tutorial folks can interact with their attendees